packages/test-helpers/src/decoding.js

```diff
@@ -40,7 +40,7 @@ async function decodeErrorReasonFromTx(tx, ctx) {
 }
 
 function decodeEvents(receipt, contractAbi, eventName) {
-  const rawLogs = receipt.rawLogs || []
+  const rawLogs = receipt.rawLogs || (receipt.receipt && receipt.receipt.rawLogs) || []
```

Comment:
Oh interesting, how do you get nested receipts?

Should we propagate this change across all of the event helpers?

Reply:
Truffle returns a nested receipt when sending transactions like `receipt = { receipt, tx }`, while web3 returns a flat receipt when calling `eth.getTransactionReceipt` for example.

Reply:
> Truffle returns a nested receipt when sending transactions like receipt = { receipt, tx }

Oh interesting, do you mean what the promise resolves to on contract invocations? This must be new truffle 5 behaviour?